app/models/miq_server/worker_management/monitor.rb

```diff
@@ -69,13 +69,17 @@ def sync_from_system
     end
 
     cleanup_orphaned_worker_rows
+
+    if podified?
```

Comment:
Do we need this since we have `return unless podified?` on https://github.com/ManageIQ/manageiq/pull/20865/files#diff-9c6cc10645dc1540242428e5ab01c51a2b0e55334f885ab9be18e50acc2e06a6R24

Reply:
Good question.  I think it's clearer that the conditional check is here.  Maybe I'll drop it from `sync_deployment_settings`

Reply:
done